src/main/java/mate.academy/Main.java

```diff
@@ -8,10 +8,14 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
-        List<Product> products = productService.getAllFromFile("products.txt");
-        products.forEach(System.out::println);
+        try {
+            ProductService productService =
+                    (ProductService) injector.getInstance(ProductService.class);
+            List<Product> products = productService.getAllFromFile("products.txt");
```

Comment:
Verify that the file path 'products.txt' is correct and accessible. If the file does not exist or the path is incorrect, this line will throw an exception.